### PR TITLE
chore(lang): improve Italian translations

### PR DESCRIPTION
**Pull request by Cristiandis**

### CODE REVIEW

**Comment by eythaann** [comment]
"Metter il fondo a" ? haha lol change that

File: src/apps/settings/i18n/translations/it.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 sides:
-  top: Superiore
+  top: Sopra
   left: Sinistra
-  bottom: Metter il fondo a
```
Comment (eythaann) [inline src/apps/settings/i18n/translations/it.yml:4]: change this lol

**Review by eythaann — CHANGES_REQUESTED** [review]

File: src/apps/settings/i18n/translations/it.yml (hunk 1 above)
Comment (Cristiandis) [reply src/apps/settings/i18n/translations/it.yml:4]: wdym? i changed that already

Comment (eythaann) [reply src/apps/settings/i18n/translations/it.yml:4]: yeah I was recently waking up hahaha sorry

**Pull request merged**